protobufs/livekit_analytics.proto

```diff
@@ -202,3 +204,28 @@ message AnalyticsNodeRooms {
   google.protobuf.Timestamp timestamp = 3;
   repeated AnalyticsRoom rooms = 4;
 }
+
+message ReportInfo {
+  oneof message {
+    FeatureUsageInfo feature_usage = 1;
+  }
+}
+
+message TimeRange {
+  google.protobuf.Timestamp started_at = 1;
+  google.protobuf.Timestamp ended_at = 2;
+}
+
+message FeatureUsageInfo {
+  enum Feature {
+    KRISP_NOISE_CANCELLATION = 0;
```

Comment:
we're going to want to meter krisp BVC and krisp NC separately as we may have different pricing arrangements for the two models, so we probably need either two features or a separate sub-feature property

Reply:
Will add another feature enum. That's the reason why I named the first one noise cancellation explicitly instead of just calling it Krisp.

Reply:
Added in this [commit](https://github.com/livekit/protocol/pull/962/commits/003d347f8ba5ba56ab959a19fd3cbee1b3ba3535).